webcurator-docs/guides/api-principles.rst

```diff
@@ -0,0 +1,39 @@
+==========
+Principles
+==========
+
+Maturity
+========
+The WCT APIs are fully RESTful webservices as defined by the REST principals (https://restfulapi.net/). They also adhere to the Richardson maturity model level two. This level of maturity makes use of URIs and HTTP Methods, but does not implement HATEOAS (https://restfulapi.net/richardson-maturity-model/).  
+
+Versioning
+==========
+For the WCT APIs the version number of the API is part of the URI.
+
+If an API is deprecated and no longer supported the HTTP response code '501: Not Implemented' will be returned. The body of the 
+reposnse will be empty.
+
+Authentication
+==============
+To use an API the user must be authenticated. This is done by getting a session token from WCT via the authentication API. This
+token must be provided with each request to an API. This is done via the adding the authorization in the request header as follows:
+Authorization: Bearer <token>
+
+<token> is the string of characters (token) returned from the authentication API.
+
+Request
+=======
+All API requests make use of HTTPS, HTTP is not supported.
```

Comment:
Something to discuss at the next meeting. How will this impact local testing/dev?

Reply:
Yeah, I think it would be a good idea to have the option to use HTTP for local testing.